vendor/github.com/prometheus/client_golang/prometheus/collectors/collectors.go

```diff
@@ -14,3 +14,27 @@
 // Package collectors provides implementations of prometheus.Collector to
 // conveniently collect process and Go-related metrics.
 package collectors
+
+import "github.com/prometheus/client_golang/prometheus"
```

Comment:
Unrelated yes, but this addresses a regression that popped up again after a master rebase. I just pulled it in here to be able to put this through CI. client_golang v1.12.1 causes the agent's API handler to panic on trying to encode NaNs as json. I'm not sure why this is working fine on master. We should definitely upgrade client_golang to v1.12.2 before cutting Cilium 1.12, since 1.12.1 greatly inflates cardinality.

This obviously needs to go in as a separate PR. :)

Reply:
Commit removed.